package.json

```diff
@@ -35,8 +35,17 @@
     "ember-cli-sentry": "3.0.0-beta.2",
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
+<<<<<<< HEAD
+<<<<<<< HEAD
     "ember-cli-template-lint": "0.7.4",
     "ember-cli-uglify": "^2.0.0",
+=======
+    "ember-cli-template-lint": "0.7.3",
+=======
+    "ember-cli-template-lint": "0.7.4",
+>>>>>>> chore(package): update ember-cli-template-lint to version 0.7.4
+    "ember-cli-uglify": "^1.2.0",
+>>>>>>> chore(package): update ember-data to version 2.16.0
```

Comment:
NIT: Some more diff text here.

Feel free to nuke `package.json` and run `npm install` on your branch.